src/oss/confluent.sh

```diff
@@ -615,15 +638,61 @@ wait_ksql() {
     wait_process_up "${pid}" 2000 || echo "ksql failed to start"
 }
 
+start_control-center() {
+    local service="control-center"
+    is_running "connect" "false" \
+        || die "Cannot start Control-Center, Kafka Connect is not running. Check your deployment"
```

Comment:
Is this what we want?  ie, if Connect hasn't started, does that mean C3 should fail?  I thought C3 could still come up...but not sure.  cc: @norwood

Reply:
connect is optional for c3

Reply:
Currently, given that the startup sequence is only one and linear this doesn't really matter. `control-center` comes after `connect` anyways, and allowing to run with connect down is not very useful. All that this means is that something was wrong with connect but C3 doesn't care. 

I'll change the dependency to be `kafka`. But again, with `confluent start` all the services should be up (and currently `confluent start control-center` is synonymous to `confluent start` with control-center being the last service in the startup order.